arq/worker.py

```diff
@@ -159,6 +160,7 @@ def __init__(
         burst: bool = False,
         on_startup: Optional['StartupShutdown'] = None,
         on_shutdown: Optional['StartupShutdown'] = None,
+        sigs: bool = True,
```

Comment:
```suggestion
        handle_signals: bool = True,
```
Bit clearer for those new to the project.